src/xargs/mod.rs

```diff
@@ -813,11 +841,24 @@ fn do_xargs(args: &[&str]) -> Result<CommandResult, XargsError> {
         )
         .arg(
             Arg::new(options::VERBOSE)
-                .short('t')
-                .long(options::VERBOSE)
-                .help("Be verbose")
-                .action(ArgAction::SetTrue),
+            .short('t')
+            .long(options::VERBOSE)
+            .help("Be verbose")
+            .action(ArgAction::SetTrue),
+        )
+        .arg(
+            Arg::new(options::REPLACE)
+                .long(options::REPLACE)
+                .short('I')
+                .short_alias('i')
+                .num_args(0..=1)
```

Comment:
I think you have to enable `require_equals`, otherwise it is not possible to figure out whether there is a value specified or not when using something like `xargs --replace echo {} bar`. And there is also a test missing to ensure that both `--replace` (using `{}` as default value) and `--replace=value` work.

The other changes look fine.

Reply:
<3 
it is now "perfect", thanks